Backend/Remora.Discord.API.Abstractions/API/Voice/Gateway/Commands/ConnectingResuming/IVoiceIdentify.cs

```diff
@@ -29,12 +29,12 @@ namespace Remora.Discord.API.Abstractions.Voice.Gateway.Commands
     /// Represents a request to identify with the voice gateway.
     /// </summary>
     [PublicAPI]
-    public interface IVoiceIdentify
+    public interface IVoiceIdentify : IVoiceGatewayCommand
     {
         /// <summary>
         /// Gets the ID of the guild to identify with.
         /// </summary>
-        Snowflake ServerID { get; }
+        Snowflake GuildID { get; }
```

Comment:
This field is documented as `server_id` in Discord's documentation. Is this correct, and if so, why?

Reply:
Supposedly, server_id seems to be a field that points to the ID of the DM, Group DM, or Guild.
Since bots can only make calls in guilds, there is no way to specify anything other than GuildID for server_id.
Therefore, I changed the field name to GuildID.

Reply:
Unless there's a style rule that says otherwise, fields should be named the same as what Discord documents and sends. While I agree that this is a more accurate name, it's not consistent with the rest of the library. Rename it to `ServerID`.

Reply:
Thanks for the decision, I'll change it back to `ServerID`.
I was wondering what the right thing to do would be, so thanks for the help.

Reply:
Revert to `ServerID` field in commit 1a6970f.